x-pack/plugins/alerts/server/usage/alerts_telemetry.ts

```diff
@@ -34,14 +35,22 @@ const alertTypeMetric = {
   },
 };
 
-export async function getTotalCountAggregations(callCluster: LegacyAPICaller, kibanaInex: string) {
+export async function getTotalCountAggregations(
+  callCluster: LegacyAPICaller,
+  kibanaInex: string
+): Promise<
+  Pick<
+    AlertsUsage,
+    'count_total' | 'count_by_type' | 'throttle_time' | 'schedule_time' | 'connectors_per_alert'
+  >
```

Comment:
Types claimed that `throttle_time` and `schedule_time` were numbers, but they are strings. Enforcing returned types to ensure we don't run into this issue in the future.